apps/vmq_server/src/vmq_reg.erl

```diff
@@ -141,6 +141,23 @@ subscribe_op(vmq_reg_redis_trie, {MP, ClientId} = SubscriberId, Topics) ->
             {ok, []} ->
                 []
         end,
+    lists:foreach(
+        fun
+            ({[<<"$share">>, _Group | Topic], QoS}) ->
+                Key = {MP, Topic},
+                Value = {ClientId, QoS},
+                T1 = vmq_util:ts(),
+                ets:insert(vmq_shared_subs_local, {{Key, Value}}),
```

Comment:
How about using a macro variable for referring the ets table? Maybe ?LOCAL_SHARED_SUBS can be used. And we can change its value to vmq_local_shared_subs.

Reply:
Don't have a strong opinion on this.

Reply:
Done